lib/datadog/tracing/contrib/active_support/configuration/settings.rb

```diff
@@ -39,6 +39,11 @@ class Settings < Contrib::Configuration::Settings
                 )
               end
             end
+
+            option :cache_key_enabled do |o|
```

Comment:
>If we stick with cache_key_enabled, and we wanted to add regex capability (for example), then we would likely have to do cache_key_exclude_pattern, cache_key_include_pattern, etc... this is OK, but idiomatically, it may be better to have a subgroup instead (e.g. cache_key.enabled, cache_key.include_pattern, cache_key.exclude_pattern.)


I like this idea, it seems much more effective and better long-term so I'll refactor to that.